<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Display2</class>
 <widget class="QDialog" name="Display2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display 2</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../datos/Escritorio/tesis/bci/Graphics/icon.png</normaloff>../../../../../../datos/Escritorio/tesis/bci/Graphics/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="GraphicsView" name="graphicsView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
